src/hotspot/share/gc/g1/g1RemSetSummary.cpp

```diff
@@ -218,15 +218,24 @@ class HRRSStatsIter: public G1HeapRegionClosure {
   bool do_heap_region(G1HeapRegion* r) {
     G1HeapRegionRemSet* hrrs = r->rem_set();
 
+    size_t occupied_cards = hrrs->occupied();
     // G1HeapRegionRemSet::mem_size() includes the
     // size of the code roots
     size_t rs_unused_mem_sz = hrrs->unused_mem_size();
     size_t rs_mem_sz = hrrs->mem_size();
+
+    if (r->is_young() && hrrs->has_group_cardset()) {
```

Comment:
When do young-regions not have group cardset? (I thought the second condition would always be true if the first is met.)

Reply:
Right, second condition should be removed.